### ISSUE TITLE
Please add multiple proxy support

### ISSUE THREAD

**Issue by he852100** [aria2/aria2#968]
Used to ease the site's speed limit for a single IP

**Comment by sachk** [comment]
Yeah, I've tried this personally by doing it manually and it works very well in some cases.
